src/input/masked-input.js

```diff
@@ -24,6 +24,9 @@ const MaskOverride = React.forwardRef<MaskedInputPropsT, HTMLElement>(
       onBlur,
       value,
       disabled,
+      // do nothing with it - we just don't want to pass it to the InputMask, as
+      // it does not have a property called positive
+      positive,
```

Comment:
do you mind co-locating this with the other 'ignored' props slightly above?

Reply:
great catch, thank you @chasestarr - just fixed it